EIPS/eip-1948.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: 1948
+title: Non-fungible Data Token
+author: Johann Barbie (@johannbarbie), Ben Bollen <ben@ost.com>, pinkiebell (@pinkiebell)
+discussions-to: https://ethereum-magicians.org/t/erc-non-fungible-data-token/3139
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-18
+requires: 721
+---
+
+## Simple Summary
+
+Some NFT use-cases require to have dynamic data associated with a non-fungible token that can change during its live-time. Examples for dynamic data:
+- cryptokitties that can change color
+- intellectual property tokens that encode rights holders
+- tokens that store data to transport them across chains
+
+The existing metadata standard does not suffice as data can only be set at minting time and not modified later.
+
+## Abstract
+
+Non-fungible tokens (NFTs) are extended with the ability to store dynamic data. A 32 bytes data field is added and a read function allows to access it. The write function allows to update it, if the caller is the owner of the token. An event is emitted every time the data updates and the previous and new value is emitted in it.
+
+## Motivation
+
+The proposal is made to standardize on tokens with dynamic data. Interactions with bridges for side-chains like xDAI or Plasma chains will profit from the ability to use such tokens. Protocols that build on data tokens like [distributed breeding](https://ethresear.ch/t/a-distributed-breeding-function/5264) will be enabled.
+
+## Specification
+
+An extension of [ERC721](https://eips.ethereum.org/EIPS/eip-721) interface with the following functions and events is suggested:
+
+```
+pragma solidity ^0.5.2;
+
+contract IERC1948 {
```

Comment:
SHouldn't this be an interface?

Reply:
thx, didn't even know the `interface` keyword :)